packages/salesforcedx-vscode-lightning/README.md

```diff
@@ -28,10 +28,10 @@ Currently, Visual Studio Code extensions are not signed or verified on the Micro
    ![List of symbols in a .js file from a Lightning bundle](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-lightning/images/lightning_outline.png)
 
 * Salesforce Lightning Design System Linter
-   * Detects deprecated BEM syntax `'--'` for Salesforce Lightning Design System class names in html files
+   * Detects deprecated BEM syntax (`--`) for Salesforce Lightning Design System class names in HTML files
    * Warning message displays on hover
-   * Code actions available on click
-   * Linter won't run if SLDS is included as a static resource in your project
+   * Code actions are available on click
+   Note: The linter won't run if SLDS is included as a static resource in your project.
 
    ![SLDS Linter detecting deprecated '--' class name syntax](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/ayesha/add-slds-linter/packages/salesforcedx-vscode-lightning/images/lightning_slds.png)
```

Comment:
We should take a new screenshot that includes the updated messages. We'll also need to update this path to use /develop/ instead of /ayesha/add-slds-linter/, but if we do that before merging in this PR the image won't display. It would be good if the resolution and text size of the screenshot approximately matched the other screenshots in this README.